src/main/ts/Part2Ex3Optional.ts

```diff
@@ -35,13 +36,19 @@ export const toPositiveInteger = (n: number): Optional<number> =>
   n > 0 ? Optional.some(n) : Optional.none();
 
 // TODO: create a function which takes a string and returns some if the string is non-empty
+export const isString = (s: string): Optional<string> => s ? Optional.some(s) : Optional.none();
```

Comment:
This will work fine, but just as a style hint we don't allow falsy checks for non-boolean values like this most times, so it should probably be something like this instead:
```suggestion
export const isString = (s: string): Optional<string> => s.length > 0 ? Optional.some(s) : Optional.none();
```